### PR TITLE
Handle chained function calls

### PR DESCRIPTION
**Pull request by rohit-jain27**

### CODE REVIEW

File: tests/test_node_visitor.py [hunk 1]
```diff
@@ -58,6 +58,18 @@ def outer_method(self):
             "nested_method_module.InnerClass.inner_method",
             {"COUNT_NESTED_METHOD_CALLS": {"nested_method_module.py": 1}},
         ),
+        (
+            "COUNT_CHAINED_FUNCTION_CALLS",
+            """
```
Comment (cabiad) [inline tests/test_node_visitor.py]: new pattern in master is to split the code out to a constant just above ...

File: tests/test_node_visitor.py [hunk 2]
```diff
@@ -76,6 +83,13 @@ class OuterClass2(object):
                 }
             },
         ),
+        (
+            "COUNT_CHAINED_FUNCTION_CALLS",
+            CHAINED_FUNCTION_CALL_CODE,
+            "function_module",
+            "function_module.function2",
```
[the remark this answers is not in the document]
Comment (rohit-jain27) [reply tests/test_node_visitor.py]: I think it's helpful to keep this one, and also add a test on `function1`. That way we can see the first one is visited successfully, but the call of the call is not being counted.

**Comment by rohit-jain27** [comment]
@cabiad added second test

File: tests/test_node_visitor.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (cabiad) [reply tests/test_node_visitor.py]: Isn't this equivalent to the first function test case? The call node is just a regular function call.

The `function1` call is the interesting one.

[the remark this answers is not in the document]
Comment (rohit-jain27) [reply tests/test_node_visitor.py]: Fair point! Updated

**Pull request merged**